### PR TITLE
Add int values to PromptBehavior enums

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

File: adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Features/NonWinCommon/PromptBehavior.cs [hunk 1]
```diff
@@ -37,27 +37,27 @@ public enum PromptBehavior
         /// Acquire token will prompt the user for credentials only when necessary.  If a token
         /// that meets the requirements is already cached then the user will not be prompted.
         /// </summary>
-        Auto,
+        Auto = 0,
```
Comment (bgavrilMS) [inline adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Features/NonWinCommon/PromptBehavior.cs:40]: This all seems to be in line with the Enum design guidelines https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/enum 

I assume the reason for doing this is to add that the default value for PromptBehaviour is Auto? If so, please add an enum comment.

**Review by bgavrilMS — APPROVED** [review]
LGTM, see minor comment inline

**Comment by henrik-me** [comment]
I assume this will not be a breaking change as likely the output would be the same. Did you validate?

**Review by henrik-me — APPROVED** [review]
:shipit:

**Comment by jennyf19** [comment]
closing as the change will be in this [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1154)

**Pull request closed without merging**